src/pages/fr/guides/deploy.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Déployer votre Site Astro
+description: Comment déployer votre site Astro sur le web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiment, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
+
+<DeployGuidesNav />
+
+## Options de déploiment rapide
+
+Vous pouvez construire et déployer rapidement un site Astro via de nombreux hébergeurs, via l'interface de leur site internet ou via une CLI.
+
+### Interface d'un hébergeur
+
+Une façon rapide de déployer votre site internet est de connecter le repository Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiment continu en utilisant Git.
+
+Ces hébergeurs détectent automatiquement les push sur le repository de votre projet Astro, build votre site et le déploient sur le web sur une URL personnalisée ou sur votre nom de domaine personnel. La plupart du temps, mettre en place le déploiment sur ces platformes suivra les étapes suivantes :
+
+1. Ajoutez votre repository à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
```

Comment:
Translation. Also we can translate 'provider' but I don't have anything good right now
```suggestion
1. Ajoutez votre dépôt à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
```